### PR TITLE
sad/edge case for single drink request from DB

### LINKED ISSUE #34: A DB Drink - Edge Case
- [x] Test the return if a field is requested twice in same query
- [x] Test for a drink ID requested that is not in DB

### PR DESCRIPTION
**Pull request by RyanChrisSmith**
## Pull request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Other (please describe):
       -  sad/edge case testing added for single drink query from the existing DB

## Problem/feature

_What problem are you trying to solve?_

_Issue Number?_
Fixes #34

## Checklist

- [x] Code compiles correctly
- [x] Tests for the changes have been added (for bug fixes/features)
- [x] All tests passing
- [ ] Extended the README / documentation, if necessary

### CODE REVIEW

File: app/graphql/re_sip_e_be_schema.rb [hunk 1]
```diff
@@ -37,4 +37,9 @@ def self.object_from_id(global_id, query_ctx)
     # For example, use Rails' GlobalID library (https://github.com/rails/globalid):
     GlobalID.find(global_id)
   end
+
+  rescue_from(ActiveRecord::RecordNotFound) do |err, obj, args, ctx, field|
+    raise GraphQL::ExecutionError, "#{field.type.unwrap.graphql_name} not found"
```
Comment (aleish-m) [inline app/graphql/re_sip_e_be_schema.rb:42]: Glad you found info on this `rescue_from` method available for GraphQL it will make Sad Path testing easy for all our DB find methods
